src/CheckoutSdk/Payments/Links/PaymentLinkDetailsResponse.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Checkout.Payments.Links
+{
+
+    public sealed class PaymentLinkDetailsResponse : Resource, IEquatable<PaymentLinkDetailsResponse>
+	{
+
+		public string Id { get; set; }
+
+		public PaymentLinkStatus? Status { get; set; }
+
+		[JsonProperty(PropertyName = "expires_on")]
+		public string ExpiresOn { get; set; }
```

Comment:
Please confirm is this can be a `DateTime`.

Reply:
This field is String on file PaymentLinkDetailsResponse.java